def test003():
>       assert 3==4
E       assert 3 == 4

testcase\test_03.py:15: AssertionError